server/chat-plugins/help.ts

```diff
@@ -142,7 +142,7 @@ export class HelpResponder {
 		return null;
 	}
 	getFaqID(faq: string) {
```

Comment:
```suggestion
	getFaqID(faq: string | undefined) {
```

It seems like this signature is wrong? And/or some function that calls it is wrong?

Reply:
It's not _supposed_ to be undefined, but apparently it could be, and this is a cleaner fix, i felt.